packages/framework/src/Resources/views/Admin/Content/Image/overview.html.twig

```diff
@@ -19,27 +19,27 @@
 {% endblock %}
 
 {% macro entitySizes(name, sizeConfigs) %}
-    <div class="wrap-border">
+    <div class="wrap-divider wrap-divider--bottom">
         <h2>{{ name|humanize }}</h2>
-        <table class="table-main">
+        <table class="table-grid">
             <thead>
-                <tr>
-                    <th class="table-col-20">{{ 'Size name'|trans }}</th>
-                    <th class="table-col-10">{{ 'Width [px]'|trans }}</th>
-                    <th class="table-col-10">{{ 'Height [px]'|trans }}</th>
-                    <th class="table-col-50">{{ 'Occurrence'|trans }}</th>
-                    <th class="table-col-10 text-right">{{ 'Cropping'|trans }}</th>
+                <tr class="table-grid__row">
+                    <th class="table-grid__cell table-col-20">{{ 'Size name'|trans }}</th>
```

Comment:
I've always thought this structure (`.table-grid > .table-grid__row > .table-grid__cell`) is also a violation of BEM. Is there some reason for it?

Reply:
Yes thats true is little bit of violation. But we are using this bcs the source code is smaller and more readable.

Specially in table where can be hundreds of elements, can be source code much bigger - 80% source code for elements, 20% for content. So in cases where it makese sence we can use it like this.

Reply:
OK, makes sense